criticality_score/run.py

```diff
@@ -639,37 +680,13 @@ def main():
 
     output = None
     if args.repo:
```

Comment:
I suggest adding an `else` condition here to handle the situation were neither `args.repo` nor `args.l_file` are present.

Reply:
Hi @calebbrown ，I've added an else to catch unknown options. I thought that the required option of the parser group was already set to true, so I didn't add an else. But you're right, it's better to add an else here.